apps/yapms/src/lib/components/modals/optionsmodal/OptionsModal.svelte

```diff
@@ -1,67 +1,83 @@
 	const chartPositionValues = ['bottom', 'left'];
+	const logos = [
+		{ name: 'Lets Talk Elections', src: LetsTalkElections },
+		{ name: 'Red Eagle Politics', src: RedEaglePolitics },
+		{ name: 'None', src: null }
+	];
 
-	function setChartType(id: string) {
-		ChartTypeStore.set(id as ChartType);
-		reapplyPanZoom();
-	}
+	$: selectedLogoName = logos.find((logo) => logo.src === $LogoStore)?.name;
 
-	function setChartPosition(id: string) {
-		ChartPositionStore.set(id as ChartPosition);
-		reapplyPanZoom();
+	function changeLogo() {
+		const logo = logos.find((logo) => logo.name === selectedLogoName);
+		if (logo !== undefined) {
+			$LogoStore = logo.src;
+		}
 	}
 </script>
 
 <ModalBase title="Options" store={OptionsModalStore}>
 	<div slot="content">
 		<div class="flex flex-col">
-			<h3 class="font-light text-lg pb-3">Chart Type</h3>
-			<div class="flex gap-3 justify-center flex-wrap">
-				{#each chartTypeValues as type}
-					<button
-						class="btn btn-lg"
-						class:btn-primary={$ChartTypeStore !== type}
-						class:btn-success={$ChartTypeStore === type}
-						on:click={() => {
-							setChartType(type);
-						}}
-					>
-						<span>
-							{type}
-						</span>
-					</button>
-				{/each}
+			<div class="lg:flex w-full">
+				<div class="form-control w-full">
+					<label class="label flex-col cursor-pointer items-start justify-start space-y-2">
+						<span class="label-text">Chart Type</span>
+						<select class="select select-bordered w-full capitalize" bind:value={$ChartTypeStore}>
+							{#each chartTypeValues as type}
+								<option>{type}</option>
+							{/each}
+						</select>
+					</label>
+				</div>
+				<div class="form-control w-full">
+					<label class="label flex-col cursor-pointer items-start justify-start space-y-2">
+						<span class="label-text">Chart Position</span>
+						<select
+							class="select select-bordered w-full capitalize"
+							bind:value={$ChartPositionStore}
+						>
+							{#each chartPositionValues as position}
+								<option>{position}</option>
+							{/each}
+						</select>
+					</label>
+				</div>
 			</div>
-			<h3 class="font-light text-lg pt-3 pb-3">Chart Position</h3>
-			<div class="flex gap-3 justify-center flex-wrap">
-				{#each chartPositionValues as type}
-					<button
-						class="btn btn-lg"
-						class:btn-primary={$ChartPositionStore !== type}
-						class:btn-success={$ChartPositionStore === type}
-						on:click={() => {
-							setChartPosition(type);
-						}}
+			<div class="form-control w-full">
+				<label class="label flex-col cursor-pointer items-start justify-start space-y-2">
+					<span class="label-text">Chart Logo</span>
+					<select
+						class="select select-bordered w-full"
+						bind:value={selectedLogoName}
+						on:change={changeLogo}
 					>
-						<span>
-							{type}
-						</span>
-					</button>
-				{/each}
+						{#each logos as logo}
+							<option>{logo.name}</option>
+						{/each}
+					</select>
```

Comment:
Is there a reason for not binding the LogoStore similar to the other selects here?

Reply:
Yeah. If we bind Logostore to value, Logostore will become the logo name and not the src like it needs to be. We could change up the store to have the logo object stored instead of just src, but I didn't do that to keep the PR small.

Reply:
I think this should solve the issue, you can set the value of the option to the source, while keeping the display value as the name.

```html
<select class="select select-bordered w-full" bind:value={$LogoStore}>
  {#each logos as logo}
    <option value={logo.src}>{logo.name}</option>
  {/each}
</select>
```

Reply:
Oh good call! Changed logos to work this way.